deploy.md

```diff
@@ -48,9 +48,10 @@ module = config.wsgi:application
 master = true
 processes = 2
 
-socket = /tmp/yunity-dev.sock
+socket = /tmp/fstool.sock
+http-socket = /tmp/fstool.http.sock
```

Comment:
yeah, found this in bash-history:
`echo -e "GET / HTTP/1.1\r\n" | sudo socat unix-connect:/tmp/yunity-dev.http.sock STDIO`

I guess I can remove it again from the deploy.md